### PR TITLE
line wraps in app/views/imports/index.html.erb now semantically relevant

### PR DESCRIPTION
**Pull request by strangrjrjr**
### What github issue is this PR for, if any?
Better line wraps!

### What changed, and why?
Followed  suggestions by @compwron to improve readability

### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### How is this tested? (please write tests!) 💖💪
See screenshots

### Screenshots please :)
**First view:**
Screen Shot 2020-09-03 at 9 24 05 AM

**Second view:**

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### CODE REVIEW

**Comment by compwron** [comment]
Your screenshots don't have any css which I think means your local setup is missing some `yarn install` but doesn't impact this PR 
Screen Shot 2020-09-03 at 10 11 55 AM
To see what this normally looks like you can log into staging https://casa-r4g-staging.herokuapp.com/ casa_admin1@example.com / 123456 :)

File: config/environments/production.rb [hunk 1]
```diff
@@ -6,12 +6,12 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = { # WARNING do not let standardrb linter change this block, it breaks
-      :address => 'smtp-relay.sendinblue.com',
```
Comment (compwron) [inline config/environments/production.rb:9]: These lines cannot be changed, please see the comment. `# WARNING do not let standardrb linter change this block, it breaks`

File: config/environments/staging.rb [hunk 2]
```diff
@@ -16,20 +16,20 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = { # WARNING do not let standardrb linter change this block, it breaks
-      :address => 'smtp-relay.sendinblue.com',
-      :port => 587,
-      :user_name => ENV["SENDINBLUE_EMAIL"],
-      :password => ENV["SENDINBLUE_PASSWORD"],
-      :authentication => 'login',
-      :enable_starttls_auto => true
+    address: "smtp-relay.sendinblue.com",
```
Comment (compwron) [inline config/environments/staging.rb:19]: These lines cannot be changed, please see the comment. `# WARNING do not let standardrb linter change this block, it breaks`

**Review by compwron — CHANGES_REQUESTED** [review]
It looks like you ran the linter also which is great :) 
One thing needs changing- the `smtp_settings` block (I put comments on it) will break (I know, it's weird) so please undo that one part of this change and then I'll approve and merge it. :)
💪💖

**Comment by strangrjrjr** [comment]
Progress by inches! 😅 I'll revert those linter changes; is there a way to exclude certain files/directories from linting? Looks like I'm having some issues with yarn; `yarn install` completes but I still don't see the updated styling.

**Comment by compwron** [comment]
Good idea! Yes it looks like files can be excluded. Please add that if you have time 👍 
https://github.com/testdouble/standard
>How do I ignore files?
>Sometimes you need to ignore additional folders or specific minified files. To do that, add a .standard.yml file to the root of your project and specify a list of files and globs that should be excluded:
```
ignore:
  - 'some/file/in/particular.rb'
  - 'a/whole/directory/**/*'
```

**Review by compwron — APPROVED** [review]
Very cool! :) ✨

**Comment by strangrjrjr** [comment]
I'll edit those tests (and google the answer myself next time) 😉

**Pull request merged**